### PR TITLE
MGS/wicket: Add support for PSC 1

### LINKED ISSUE #6993: wicket / MGS: add support for PSC 1

### PR DESCRIPTION
**Pull request by jgallagher**
Need to test this on a rack with two PSCs before merging, but running wicket locally I think this updates all the spots where the UI needs to show PSC 1 and allow navigating to it.

Closes #6993.

### CODE REVIEW

**Review by andrewjstone — APPROVED** [review]
Looks good! Thanks @jgallagher.

File: wicket/src/state/rack.rs [hunk 1]
```diff
@@ -128,9 +125,8 @@ impl RackState {
             ComponentId::Sled(16) => ComponentId::Switch(1),
             ComponentId::Sled(0) => ComponentId::Sled(31),
             ComponentId::Sled(i) => ComponentId::Sled(i - 1),
-            // Skip over Psc(1) because it is always empty in currently shipping
-            // racks.
-            ComponentId::Switch(1) => ComponentId::Psc(0),
+            ComponentId::Switch(1) => ComponentId::Psc(1),
+            ComponentId::Psc(1) => ComponentId::Psc(0),
```
Comment (sunshowers) [inline wicket/src/state/rack.rs:129]: Do we have a test to ensure that `next` and `prev` cancel each other out? Probably valuable to have that.

File: wicket/src/ui/widgets/rack.rs [hunk 2]
```diff
@@ -147,8 +147,7 @@ impl<'a> Rack<'a> {
         let inner = block.inner(power_shelf);
         block.render(power_shelf, buf);
 
-        if i == 0 && presence == ComponentPresence::Present {
-            // Shipping racks only have one power shelf -- only show that one.
+        if presence == ComponentPresence::Present {
```
[the remark this answers is not in the document]
Comment (jgallagher) [reply wicket/src/ui/widgets/rack.rs:150]: I think `resize` is okay; we already show a row for the second PSC, but it was always blank: https://github.com/oxidecomputer/omicron/blob/41b78f172bf5d10429f34be107238056300878cb/wicket/src/ui/widgets/rack.rs#L259-L262

**Comment by jgallagher** [comment]
Tested this on a rack with two PSCs and confirmed we could see both of them. Will merge once we're past the CI test flakes.

**Pull request merged**